python/cudf_polars/cudf_polars/dsl/expr.py

```diff
@@ -867,7 +867,14 @@ def __init__(
         self.name = name
         self.options = options
         self.children = children
-        if self.name not in ("mask_nans", "round", "setsorted", "unique"):
+        if self.name not in (
+            "mask_nans",
+            "round",
+            "setsorted",
+            "unique",
+            "dropnull",
+            "fill_null",
```

Comment:
Why is `dropnull` without an underscore while `fill_null` has one... seems like something to request alignment on in polars.

Reply:
Agreed.

Reply:
This can be fixed when we update the expression node on the rust side, which it seems like we need to do anyways.

Reply:
Just to note that if we do that we need to map the old name to the new name until such time as we drop support for the older versions (probably do this in `translate.py`)